src/lib/utils/SynapseConstants.ts

```diff
@@ -118,3 +120,9 @@ export const ENTITY_HEADER_STORAGE_KEY = 'INFO_FROM_IDS_ENTITY_HEADER'
 
 export const DATETIME_UTC_COOKIE_KEY =
   'org.sagebionetworks.synapse.datetime.utc'
+
+export const BASE = 1024,
```

Comment:
Looks like you started to think about the file size restriction, but I don't think it's mentioned in this PR

Reply:
Related to a new ticket - https://sagebionetworks.jira.com/browse/SWC-6291
I'd like to use the SRC HtmlPreview in SWC

Reply:
No reason we couldn't export this and use it there. The file size limit shouldn't have been added here but it will be in the next PR.